files-override/shared/app/router.js

```diff
@@ -0,0 +1,9 @@
+import EmberRouter from '@embroider/router';
+import config from 'my-fancy-app/config/environment';
```

Comment:
This needs to use the template variable for the app name rather than hard coding "my fancy app" 🤣

Reply:
yes, fixed, thanks :)